model/src/main/java/org/projectnessie/model/MergeResponse.java

```diff
@@ -59,14 +59,11 @@ default boolean wasSuccessful() {
   @Nullable
   String getExpectedHash();
 
-  /** List of commit-IDs to be merged or transplanted. */
+  @Deprecated // for removal and replaced with something else
```

Comment:
Could we also add deprecation flag to the OpenAPI via `@Schema` annotations (cf. `RefLogResponse`)?

Reply:
Duh - somehow I thought you did that in the v2 PR